src/rest/remote-api.ts

```diff
@@ -154,7 +154,7 @@ export interface IRemoteAPI {
   /**
    * Return list of registered oAuth providers.
    */
-  getOAuthProviders(): Promise<string[]>;
+  getOAuthProviders(): Promise<{ name: string,  endpointUrl: string }[]>;
```

Comment:
Can you extract the return type as an interface?

```js
export interface IOAuthProvider {
  name: string;
  endpointUrl: string;
}
```

```suggestion
  getOAuthProviders(): Promise<IOAuthProvider[]>;
```

Reply:
I have fixed it.